test/components/user_menu/signed_in_component_test.rb

```diff
@@ -60,6 +60,46 @@ class UserMenu::SignedInComponentTest < ViewComponent::TestCase
     assert_no_link_to admin_conversations_path
   end
 
+  test "links to new notifications view when no new notifications exist" do
+    user = users(:with_business)
+
+    render_inline UserMenu::SignedInComponent.new(user)
+    assert_link_to notifications_path
+  end
+
+  test "links to new notifications view when new notifications exist" do
+    user = users(:with_business)
+    developer = developers(:available)
+    Message.create!(developer: developer, business: user.business, sender: developer, body: "Hello!")
+
+    render_inline UserMenu::SignedInComponent.new(user)
+    assert_link_to notifications_path
+  end
+
+  test "shows red alert dot when new notifications exist" do
+    user = users(:with_business)
+    developer = developers(:available)
+    Message.create!(developer: developer, business: user.business, sender: developer, body: "Hello!")
+
+    render_inline UserMenu::SignedInComponent.new(user)
+    assert_element "notification-alert"
+  end
+
+  test "does not show red alert dot when no new notifications exist" do
```

Comment:
This test tests that no notification exists at all, it doesn't test that there is a notification but it is _read_.

Reply:
Going to hold off on this until I get my head straight about the testing suite.